core-feature-pack/common/src/main/resources/content/bin/common.ps1

```diff
@@ -375,57 +375,14 @@ Function Start-WildFly-Process {
 
 # Returns java -version output for JDK defined by java opts
 Function Get-Java-Version ($javaOpts) {
-    if ($javaOpts -match '-d64') {
-        $opt_version = '-d64'
-    } elseif ($javaOpts -match '-d32') {
-        $opt_version = '-d32'
-    }
-
-    $result = & $JAVA $opt_version -version 2>&1
+    $result = & $JAVA -version 2>&1
     if ($LASTEXITCODE -eq 1) {
         $result = & $JAVA -version 2>&1
     }
 
     return $result
 }
```

Comment:
This function can be completely removed since it is no longer used by WildFly scripts.

Otherwise, it would need some changes, since now the `$javaOpts` argument is unused by the function, so it is no longer doing what it was expected to do. In addition, now the error if-check is redundant since it repeats the same command that generated the error.

We can safely remove it.

Reply:
Thanks, @yersan . Removed, squashed, and re-pushed. Sorry for needing multiple iterations. 
Remove `-server`/`-client` and `-d32`/`-d64` logic from scripts